support-frontend/app/controllers/IdentityController.scala

```diff
@@ -81,6 +83,27 @@ class IdentityController(
         )
     }
   }
+
+  def getUser(): Action[AnyContent] = maybeAuthenticatedAction().async { implicit request => {
+    type Attempt[A] = EitherT[Future, String, A]
+    request.user.traverse[Attempt, IdUser](identityService.getUser(_))
+      .fold(
+        _ => {
```

Comment:
what's the underscore? should it be logged? and should it be a warn rather than an info?